rclcpp/include/rclcpp/function_traits.hpp

```diff
@@ -130,20 +130,20 @@ struct function_traits<FunctionT &&>: function_traits<FunctionT>
  */
 template<std::size_t Arity, typename FunctorT>
 struct arity_comparator : std::integral_constant<
-    bool, (Arity == function_traits<FunctorT>::arity)>{};
+    bool, (Arity == function_traits<FunctorT>::arity)> {};
 
 template<typename FunctorT, typename ... Args>
 struct check_arguments : std::is_same<
     typename function_traits<FunctorT>::arguments,
     std::tuple<Args ...>
-  >
+>
```

Comment:
This one has the right indentation for the closing `>`, but does 4 spaces rather than two on the inside... Which is somehow the counterpart of the one I commented on above. If only we could combine the two...